Formula/p/prjtrellis.rb

```diff
@@ -0,0 +1,48 @@
+class Prjtrellis < Formula
+  desc "Documenting the Lattice ECP5 bit-stream format"
+  homepage "https://github.com/SymbiFlow/prjtrellis"
```

Comment:
Homepage is pointing to a fork.
```suggestion
  homepage "https://github.com/YosysHQ/prjtrellis"
```

Reply:
👍

Reply:
good catch!